plugins/filtering/src/__snapshots__/index.test.js.snap

```diff
@@ -2,12 +2,24 @@
 
 exports[`plugin in a stock JBrowse 1`] = `
 Object {
+  "twoBitLocation": Object {
```

Comment:
is it expected that snapshots are changing like this? seems like it might be good to not have that happen

Reply:
My assumption is that if the model changes, the snapshots will change? Can I suppress what ends up in the snapshot?